src/packs/nzism-v36-1022-20.ts

```diff
@@ -0,0 +1,845 @@
+      explanation:
+        'MUST 22.1.23.C.01[CID:4829]| Enterprise systems security/Cloud Computing/System Availability',
+      level: NagMessageLevel.ERROR,
+      rule: RDSMultiAZSupport,
+      node: node,
+    });
+
+    this.applyRule({
+      info: 'MUST 22.1.26.C.01[CID:4849]| Enterprise systems security/Cloud Computing/Backup, Recovery Archiving and Data Remanence',
+      explanation:
+        'To help with data back-up processes, ensure your Amazon Relational Database Service (Amazon RDS) instances are a part of an AWS Backup plan. AWS Backup is a fully managed backup service with a policy-based backup solution. This solution simplifies your backup management and enables you to meet your business and regulatory backup compliance requirements.',
+      level: NagMessageLevel.ERROR,
+      rule: RDSInBackupPlan,
+      node: node,
+    });
+
+    this.applyRule({
+      info: 'MUST 19.1.12.C.01[CID:3562], SHOULD 19.1.14.C.02[CID:3623], MUST 23.4.10.C.01[CID:7466]| Gateway security/Gateways: (Configuration of gateways & Demilitarised zones) & Public Cloud Security/Data Protection in Public Cloud/Data accessibility',
+      explanation:
+        'Amazon RDS database instances can contain sensitive information, and principles and access control is required for such accounts.',
+      level: NagMessageLevel.ERROR,
+      rule: RDSInstancePublicAccess,
+      node: node,
+    });
+
+    this.applyRule({
+      info: 'SHOULD 16.6.10.C.02[CID:2013], SHOULD 20.4.4.C.02[CID:4441], SHOULD 20.4.5.C.02[CID:4445], MUST 23.5.11.C.01[CID:7496]| Access Control and Passwords/Event Logging and Auditing/Additional events to be logged & Data management/Databases: (Database file...',
+      explanation:
+        'To help with logging and monitoring within your environment, ensure Amazon Relational Database Service (Amazon RDS) logging is enabled. With Amazon RDS logging, you can capture events such as connections, disconnections, queries, or tables queried.' +
+        "This is a granular rule that returns individual findings that can be suppressed with 'appliesTo'. The findings are in the format 'LogExport::<log>' for exported logs. Example: appliesTo: ['LogExport::audit'].",
+      level: NagMessageLevel.ERROR,
+      rule: RDSLoggingEnabled,
+      node: node,
+    });
+
+    this.applyRule({
+      info: 'SHOULD 17.1.46.C.04[CID:2082], SHOULD 20.4.4.C.02[CID:4441], SHOULD 22.1.24.C.04[CID:4839]| Cryptography/Cryptographic Fundamentals/Reducing storage and physical transfer requirements & Data management/Databases/Database files & Enterprise systems se..',
+      explanation:
+        'Because sensitive data can exist at rest in Amazon RDS instances, enable encryption at rest to help protect that data.',
+      level: NagMessageLevel.ERROR,
+      rule: RDSStorageEncrypted,
+      node: node,
+    });
+
+    // below this not checked
```

Comment:
Is there a reason for this?

Reply:
That was a note to myself while i was constructing it, that should have got deleted before i submitted it.